canonical/multipass
# Release 1.7.0

Test platforms:
- [x] Ubuntu@qemu (Luis)
- [x] Ubuntu@LXD (Chris)
- [x] Ubuntu@libvirt (Chris)
- [x] macOS@hyperkit (Chris)
- [x] macOS@VirtualBox (Chris)
- [x] Windows@Hyper-V (Luis)
- [x] Windows@VirtualBox (Luis)

Test plan:
1. [install previous release](https://multipass.run/#install)
2. launch a couple instances (including `primary`) and stop/suspend them
3. update to the RC, verify instances around and in the expected state
4. uninstall / purge and install new
5. launch instances (with, and without `--network`, `--bridged`, verify bridge creation on LXD and Hyper-V)
   - `primary`
   - older Ubuntu
   - Ubuntu Core
   - file / http
   - workflows (`minikube`)
